capa/render/__init__.py

```diff
@@ -163,11 +163,44 @@ def convert_match_to_result_document(rules, capabilities, result):
 
 
 def convert_meta_to_result_document(meta):
+    attacks = meta.get("att&ck", [])
+    meta["att&ck"] = [parse_canonical_attack(attack) for attack in attacks]
     mbcs = meta.get("mbc", [])
     meta["mbc"] = [parse_canonical_mbc(mbc) for mbc in mbcs]
     return meta
 
 
+def parse_canonical_attack(attck):
```

Comment:
some code dup here but i think thats just fine. these code is quite readable and unlikely to change often.

Reply:
improved in 63cd700